### ISSUE TITLE
Bar chart fillcolor not working

### ISSUE THREAD

**Issue by MBFouad** [PHPOffice/PhpSpreadsheet#1400]
This is:

```
- [* ] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
If I set the colors for a BarChar, I'm expecting the lines to have those colors.

### What is the current behavior?
BarChar don't have those colors

### What are the steps to reproduce?
Take the sample file 33_Chart_create_bar_stacked.php, and add a color.

just change this line 
 $dataSeriesValues = array(
                    new DataSeriesValues(DataSeriesValues::DATASERIES_TYPE_NUMBER, 'Worksheet!$D$2:$D$5', NULL, 4, array(), null, '000000')) ;

### Which versions of PhpSpreadsheet and PHP are affected?
 the latest

**Comment by MBFouad** [comment]
I create pull request #1401

**Comment by oleibman** [comment]
Fixed by PR #2906 in June 2022.
